app/views/tags/index.html.erb

```diff
@@ -16,12 +16,19 @@
         <thead>
           <tr class="table-header">
             <th>Name</th>
+            <th></th>
```

Comment:
I've been putting 'Actions' in here as the header - it's wise to have a header for accessibility even if you wrap it in span with a sr-only class to only show it to screen readers.